pelican/plugins/image_process/image_process.py

```diff
@@ -260,7 +262,7 @@ def apply_filter(i, f):
 
 
 def harvest_images(path, context):
-    log.debug("process_images: harvest %r", path)
+    log.debug("[image_process] process_images: harvest %r", path)
```

Comment:
Strip out the old logging prefix

Reply:
On this line only?

As a Pelican user, I found quite useful to know from which plugin a warning log message comes from,
would you agree to preserve this "prefix" for calls to `log.warn` ?

Reply:
The `log.debug` is fine. Edit this part:

[image_process] <strike>process_images:</strike>

Reply:
Done ;)